public/js/apps/management/app.jsx

```diff
@@ -93,28 +101,35 @@ export default class ManagementApp extends Component {
           {...boundPayMethodActions}
         />
       ));
+    } else if (connector.management.view === 'SHOW_SUBSCRIPTIONS') {
+      children.push((
+        <Subscriptions
+          {...boundSubscriptionActions}
+          userSubscriptions={connector.user.subscriptions}
+        />
+      ));
+    } else if (connector.management.view === 'SHOW_HISTORY') {
+      children.push((
+        <History />
+      ));
     }
```

Comment:
We will soon have a default view (sign-in) so I'm wondering if this would work best as a big switch statement.